src/Phaser.C

```diff
@@ -76,7 +82,7 @@ Phaser::out (float * smpsl, float * smpsr)
         rgain = 0.0f;
 
     for (i = 0; i < period; i++) {
-        float x = (float) i / fPERIOD;
+        float x = (float) i / float(period);
```

Comment:
yeah, there's a ton of stylistic discrepancies throughout the code. It could really use some TLC. My goal though was to change the least amount possible and get it to function safely as a plugin. The next step was to try to improve everything but I've been working on other projects. I still intend to come back to these, but you can go ahead and make whatever improvements you like in the mean time.